microsoft-edge/devtools-guide-chromium/whats-new/2023/04/devtools-112.md

```diff
@@ -0,0 +1,180 @@
+
+![The unminified call stack from a profile recorded in the Performance tool still showing minified names](./devtools-112-images/unminification-in-oopifs-before.png)
+
+In Microsoft Edge 112, the **Performance** tool now loads sourcemaps and applies them to the performance events that are tracked in OOPIFs.  As a result, after recording a profile in the **Performance** tool, you can now see unminified file and function names for the JavaScript execution that occurs within an OOPIF:
+
+![The unminified call stack from a profile recorded in the Performance tool](./devtools-112-images/unminification-in-oopifs-after.png)
+
+See also:
+* [Performance tool unminifies file and function names when profiling production code](./devtools-111.md#performance-tool-unminifies-file-and-function-names-when-profiling-production-code) in _What's new in DevTools (Microsoft Edge 111)_.
+* [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](../../../javascript/publish-source-maps-to-azure.md)
+* [Securely debug original code by using Azure Artifacts symbol server source maps](../../../javascript/consume-source-maps-from-azure.md)
+* [Record performance](../../../evaluate-performance/reference.md#record-performance) in _Performance features reference_.
+
+
+<!-- ====================================================================== -->
+## CSS Overview tool displays non-simple selectors for quick performance gains
+
+<!-- Subtitle: Non-simple selectors can be common culprits for long-running Recalculate Styles events in the Performance tool. Take a quick snapshot in the CSS Overview tool to see your non-simple selectors. -->
+
+The **CSS Overview** tool has a new **Non-simple selectors** section, which displays a list of non-simple CSS selectors when you take an overview snapshot of your webpage's CSS.  This list of non-simple CSS selectors in the **CSS Overview** tool provides a quick way to identify common culprits that might be causing long-running **Recalculate Styles** events:
+
+![The new, Non-simple selectors list in the CSS Overview tool](./devtools-112-images/non-simple-selectors-in-css-overview.png)
+
+This is a list of selectors that could possibly reduce performance.  Use the **Performance** tool's Selector Stats feature to determine whether these selectors are actually reducing performance.
+
+You can try to change these selectors so that they are more specific and match fewer elements, reducing the amount of time the browser takes to calculate styles in the following cases:
+
+- When elements are added to or removed from the DOM.
+- When an element's attributes are changed, such as `class` or `id`.
+- When user input occurs, such as a mouse move or a change of focus, which can affect `:hover` rules.
+
+To analyze individual CSS selector performance and analyze the impact of your changes, use the **Selector Stats** feature in the **Performance** tool, as follows:
```

Comment:
I think everything from this point on in the section is extraneous. We've already announced Selector Stats in a previous What's New and it's a separate but related feature to this announcement. I would cut everything between this line and the **See also:**

Reply:
removed locally